doc/source/install.rst

```diff
@@ -251,36 +249,51 @@ Starting Data Collection
 After you install *PMM Client*,
 enable data collection using the ``pmm-admin`` tool:
 
-1. To enable general system metrics monitoring,
-   run ``pmm-admin add os`` followed by the IP address
-   of the *PMM Client* host. For example:
+To enable general system metrics monitoring,
+run ``pmm-admin add os`` followed by the IP address
+of the *PMM Client* host. For example:
 
-   .. prompt:: bash
+.. prompt:: bash
+
+   sudo pmm-admin add os 192.168.100.2
+
+MySQL Data
+**********
 
-      sudo pmm-admin add os 192.168.100.2
+To enable MySQL metrics monitoring and query analytics,
+run ``pmm-admin add mysql``.
 
-2. To enable MySQL metrics monitoring and query analytics,
-   run ``pmm-admin add mysql``.
+.. note:: Query analytics must be able to detect
+   the local MySQL instance and MySQL superuser credentials.
+   Make sure that the necessary options are specified
+   in :file:`~/.my.cnf`. For example:
+
+   .. code-block:: none
+
+      user=root
+      password=pass
+      socket=/var/run/mysqld/mysqld.sock
+
+   Alternatively, you can specify MySQL superuser credentials
+   as command-line options for the ``pmm-admin`` tool:
+
+   .. prompt:: bash
 
-   .. note:: Query analytics must be able to detect
-      the local MySQL instance and MySQL superuser credentials.
-      Make sure that the necessary options are specified
-      in :file:`~/.my.cnf`. For example:
+      pmm-admin -user root -password pass add mysql
 
-      .. code-block:: none
+For a complete list of command-line options, run ``pmm-admin -help``.
 
-         user=root
-         password=pass
-         socket=/var/run/mysqld/mysqld.sock
+MongoDB Data
+************
 
-      Alternatively, you can specify MySQL superuser credentials
-      as command-line options for the ``pmm-admin`` tool:
+To enable MongoDB metrics monitoring, run ``pmm-admin add mongodb``.
 
-      .. prompt:: bash
+You can use options to specify the MongoDB replica set, cluster name,
+and node type. For example:
 
-         pmm-admin -user root -password pass add mysql
+.. code-block::
 
-   For a complete list of command-line options, run ``pmm-admin -help``.
+   pmm-admin -mongodb-replset repl1 -mongodb-cluster cluster1 -mongodb-nodetype mongod add mongodb
```

Comment:
Below, `pmm-admin list` output is different. Now it has less columns. Can we update it as well?